core/drand_beacon_control.go

```diff
@@ -826,6 +836,8 @@ func (bp *BeaconProcess) Status(c context.Context, in *drand.StatusRequest) (*dr
 	bp.state.Lock()
 	defer bp.state.Unlock()
 
+	bp.log.Debugw("Processing incoming Status request", "")
```

Comment:
does the empty string have a meaning here?

Reply:
Good catch! That's unfinished.